src/index.ts

```diff
@@ -120,7 +163,7 @@ export default {
         await octokit.rest.issues.removeLabel({
           ...REPO_INFO,
           issue_number: pr.pull_request.number,
-          name: label,
+          name: label as string,
```

Comment:
This was coming back as `unknown` unless you can think of a better way to type this